formats/policy/frontend/schema.json

```diff
@@ -99,6 +99,9 @@
                     }
                   }
                 }
+              },
+              "result_metadata_name_key": {
```

Comment:
I totally misunderstand, thought you mean the schema the policy publisher publishes.

Makes sense now, i'll add that.